src/components/Search/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import './styles.scss';
+
+export default function Search() {
```

Comment:
```suggestion
export default function HomePageSearch() {
```

Since there could be another non-reusable search component, Do you think we should reword to something more specific?

Reply:
I agree.